external/fv3fit/fv3fit/_shared/pytorch/Graphloss.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import dataclasses
+
+
+def _standard_mse():
+    loss = torch.nn.MSELoss()
+
+    def custom_loss(y_true, y_pred):
+        return loss(y_pred, y_true)
+
+    return custom_loss
+
+
+def _standard_mae():
+    loss = torch.nn.L1Loss()
+
+    def custom_loss(y_true, y_pred):
+        return loss(y_true, y_pred)
+
+    return custom_loss
+
+
+@dataclasses.dataclass
+class LossConfig:
+    """
+    Attributes:
+        loss_type: one of "mse" or "mae"
+        multistep: number of successive loss calculation before each optimization
+    """
+
+    multistep: int = 1
```

Comment:
`multistep` does not appear to control anything to do with the loss. Should this attribute exist on your optimizer or training loop config, instead? If so please delete it from here.